src/components/SocialLocationPreview/SocialLocationPreview.tsx

```diff
@@ -43,7 +43,7 @@ const SocialLocationPreview = (props: {
         <AlarmLevel color={fillColor}>{levelInfo.name}</AlarmLevel>
       </PreviewHeader>
       <PreviewBody>
-        <SummaryStats stats={props.stats} condensed={true} />
```

Comment:
`SummaryStats` was used here and in Embed.tsx, but `condensed` is true in both places, so now we always show the "condensed" version.